Tests/DependencyInjection/FlowsBuilderCompilerPassTest.php

```diff
@@ -95,12 +97,11 @@ public function testProcess()
             ->method('findTaggedServiceIds')
             ->willReturnMap(
                 [
-                    [FlowsBuilderCompilerPass::TAG_DEFINITIONS, ['serviceId' => [['nodeName' => 'abc']]]],
-                    ['producer.direct.demo', $serviceId],
-                    ['producer.custom.business_demo', $serviceId],
+                    [FlowsBuilderCompilerPass::TAG_DEFINITIONS, null, null, ['serviceId' => [['nodeName' => 'abc']]]],
```

Comment:
Same here. Why nullable parameters to be mocked?

Reply:
As explained in the other comment. SF2.8 does not require all map parameters but SF3.4 does.